docs/src/main/paradox/docs/getting-started/try-nexus.md

```diff
@@ -1,190 +1,704 @@
 # Try Nexus
 
-In this tutorial, you will use the core features of the Nexus ecosystem through our sandbox. This requires minimal technical knowledge but the ability to install a Python library and run a jupyter notebook.
+## Step 0: Python Setup
 
-In the @ref:[first step](try-nexus.md#configuring-your-project-in-nexus-fusion), you'll learn:
+In this tutorial, you are encouraged to run Nexus Forge to interact with Nexus. Nexus Forge is a python library. We provide Python Notebooks with examples.
```

Comment:
Maybe use "Jupyter notebooks" instead of  "Python notebooks" since they are not python specific